### PR TITLE
Update bundled theme changelog and asset version for 6.4 release

### PR DESCRIPTION
**Pull request by kafleg**
Update version, tested upto, changelog and CSS version
Core ticket: https://core.trac.wordpress.org/ticket/58832

### CODE REVIEW

File: src/wp-content/themes/twentyten/functions.php [hunk 1]
```diff
@@ -764,7 +764,7 @@ function twentyten_widget_tag_cloud_args( $args ) {
  */
 function twentyten_scripts_styles() {
 	// Theme block stylesheet.
-	wp_enqueue_style( 'twentyten-block-style', get_template_directory_uri() . '/blocks.css', array(), '20230627' );
+	wp_enqueue_style( 'twentyten-block-style', get_template_directory_uri() . '/blocks.css', array(), '20231107' );
```
Comment (sabernhardt) [inline src/wp-content/themes/twentyten/functions.php:767]: Most stylesheets were not updated in this release cycle, so only ten would need new version strings.
- Update the `style.css` files for themes from Twenty Ten to Twenty Seventeen to `20231107`.
- Update the `blocks.css` files for Twenty Thirteen and Twenty Sixteen to `20231016`.

Comment (kafleg) [reply src/wp-content/themes/twentyten/functions.php:767]: Okay, I will only change the style.css version for themes from 2010 - 2017 and blocks.css for 2013 and 2016

**Comment by kafleg** [comment]
@sabernhardt I found small versioning issue in 2019 theme. 

The current version of theme in org is 2.5 but I found that the version 2.6 is already released according to readme file and mentioned version, https://codex.wordpress.org/Twenty_Nineteen_Theme_Changelog#Version_2.6 

Would you please give a look on it?

**Comment by sabernhardt** [comment]
All 13 themes were [updated in August](https://core.trac.wordpress.org/changeset/56315). However, half of them are a version behind that at [wordpress.org/themes](https://wordpress.org/themes/).
- [Twenty Sixteen](https://wordpress.org/themes/twentysixteen/)
- [Twenty Seventeen](https://wordpress.org/themes/twentyseventeen/)
- [Twenty Nineteen](https://wordpress.org/themes/twentynineteen/)
- [Twenty Twenty](https://wordpress.org/themes/twentytwenty/)
- [Twenty Twenty-One](https://wordpress.org/themes/twentytwentyone/)
- [Twenty Twenty-Two](https://wordpress.org/themes/twentytwentytwo/)

Also note that Twenty Twenty _might_ still have a version 2.4 this month and then 2.5 with the other themes in November ([Trac 59576](https://core.trac.wordpress.org/ticket/59576)). The special effort of a separate update becomes less valuable each day, though.

**Comment by kafleg** [comment]
> All 13 themes were [updated in August](https://core.trac.wordpress.org/changeset/56315). However, half of them are a version behind that at [wordpress.org/themes](https://wordpress.org/themes/).
> 
> * [Twenty Sixteen](https://wordpress.org/themes/twentysixteen/)
> * [Twenty Seventeen](https://wordpress.org/themes/twentyseventeen/)
> * [Twenty Nineteen](https://wordpress.org/themes/twentynineteen/)
> * [Twenty Twenty](https://wordpress.org/themes/twentytwenty/)
> * [Twenty Twenty-One](https://wordpress.org/themes/twentytwentyone/)
> * [Twenty Twenty-Two](https://wordpress.org/themes/twentytwentytwo/)
> 
> Also note that Twenty Twenty _might_ still have a version 2.4 this month and then 2.5 with the other themes in November ([Trac 59576](https://core.trac.wordpress.org/ticket/59576)). The special effort of a separate update becomes less valuable each day, though.

What can I do in this case? Can i update the theme version based on the trunk version?

**Comment by sabernhardt** [comment]
@desrosj should be involved in this conversation too. My advice would be to continue updating the version according to trunk.

I would like to know whether Twenty Twenty will have a separate version to fix the iframe editor background color (which was backported to 6.3). If we delay the change three more weeks instead, I could close [59576](https://core.trac.wordpress.org/ticket/59576) as a duplicate of [58832](https://core.trac.wordpress.org/ticket/58832) and change the milestone on [59086](https://core.trac.wordpress.org/ticket/59086) to 6.4. My patch for Twenty Twenty does not (yet) update the "Tested up to" version, but that might be almost as appropriate if released this week as it would be November 7.

**Comment by desrosj** [comment]
Thanks all for working on this!

I've gone back and tried to find a reason why those themes were not uploaded on 6.3 release day and I couldn't find any. I have gone back and uploaded the updates for all of those that were missing in the state that they were on release day. This includes Twenty Twenty without the iframe changes.

I think it's best to just include the iframe change in 2 weeks with the WP 6.4 coordinated release, which will now be 2.4.

I still need to give a true review to this PR, but wanted to update my progress so far. Will circle back in the next 48 hours to do that, though I think there may be a few updates that need to be added for Twenty Twenty now as well.

**Comment by sabernhardt** [comment]
I made a new [PR 5601](https://github.com/WordPress/wordpress-develop/pull/5601) that includes Twenty Nineteen to Twenty Twenty-Three, plus it corrects the changelog link for Twenty Twenty-Four.

**Comment by desrosj** [comment]
This was handled in https://core.trac.wordpress.org/changeset/57069. I missed this connected PR when committing but @kafleg I've manually added you as a contributor to the change, though!

**Pull request closed without merging**